GiniVision/Classes/Core/Multipage Review/MultipageReviewViewController.swift

```diff
@@ -72,6 +72,19 @@ public final class MultipageReviewViewController: UIViewController {
         return view
     }()
     
+    lazy var pagesCollectionBottomTipLabel: UITextView = {
+        let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.text = NSLocalizedString("ginivision.multipagereview.dragAndDropTip",
+                                       bundle: Bundle(for: GiniVision.self),
+                                       comment: "drag and drop tip shown below pages collection")
+        textView.font = textView.font?.withSize(11)
```

Comment:
That's what I was talking about in the last PR. Now the font is 11, not 12. I believe a consistent size would have been a little better.

Reply:
As I said in the previous PR, I will extract this values to a class in a different PR.